e2e/terraform/Makefile

```diff
@@ -1,4 +1,4 @@
-NOMAD_SHA := $(shell git rev-parse HEAD)
+NOMAD_SHA ?= $(shell git rev-parse HEAD)
```

Comment:
🎉

Reply:
TIL - I've been using `make FOO=BAR target` so didn't notice this issue much before.  Thanks!